tsdb/engine/tsm1/reader.go

```diff
@@ -391,6 +405,10 @@ func (d *indirectIndex) Add(key string, blockType byte, minTime, maxTime time.Ti
 	panic("unsupported operation")
 }
 
+func (d *indirectIndex) Write(w io.Writer) error {
+	panic("unsupported operation")
```

Comment:
Yeah.  It's should never be called because the `indirectIndex` does not support writing.  I have to satisfy the `TSMIndex` interface though.